zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -0,0 +1,556 @@
+            return None
+
+        try:
+            d = jsonobject.loads(base64.b64decode(redundance_and_placement_policy))
+            return self.RedundanceAndPlacementPolicy(
+                copyset_number=d.copysetNum,
+                replica_number=d.replicaNum,
+                zone_number=d.zoneNum
+            )
+        except Exception as e:
+            logger.error('failed to decode redundance and placement policy[%s], error[%s]' % (
+                redundance_and_placement_policy, e.message
```

Pylint (3.3.7)

[error] Instance of 'Exception' has no 'message' member

(E1101)